bundles/org.openhab.binding.smartthings/contrib/smartthings/SmartApps/OpenHabAppV2.groovy

```diff
@@ -865,15 +867,23 @@ def actionColorControl(device, attribute, value) {
             device.setSaturation(value as int)
         break
         case "color":
-            def colormap = ["hue": value[0] as int, "saturation": value[1] as int]
-            // log.debug "actionColor: Setting device \"${device}\" with attribute \"${attribute}\" to colormap \"${colormap}\""
-            device.setColor(colormap)
-            device.setLevel(value[2] as int)
+    		if (value == "off") {
```

Comment:
Don't compare strings with `==`:
```suggestion
    		if ("off".equals(value)) {
```

Reply:
In Groovy == is shorthand for .equals() See groovv docs [here](https://docs.groovy-lang.org/latest/html/documentation/core-operators.html#_relational_operators)

Reply:
Ah ok. I just read it as java code :smile: I just noticed the code isn't auto formatted as it's no java code. You might replace the tabs with spaces.